deploy/charts/openebs-monitoring/values.yaml

```diff
@@ -188,6 +188,8 @@ customDashboards:
     enabled: true
   LocalPV:
     enabled: true
+  LVM:
```

Comment:
Can we rename the directory to `lvm-LocalPV`?

Reply:
Changing it to LocalPV-lvm. We can add other flavours also under LocalPv-<localpv flavours> dir.